### ISSUE TITLE
[bug] cli cannot ignore fk

### ISSUE THREAD

**Issue by lookfirst** [mikro-orm/mikro-orm#186]
b20

```
mikro-orm schema:drop --no-fk

Drop database schema based on current metadata

Options:
  -r, --run      Runs queries                                          [boolean]
  -d, --dump     Dumps all queries to console                          [boolean]
  --no-fk        Disable foreign key checks if possible[boolean] [default: true]
  -v, --version  Show version number                                   [boolean]
  -h, --help     Show help                                             [boolean]

Unknown argument: fk
```

**Comment by B4nan** [comment]
Hmm strange, `yargs` don't like the hyphen I guess. I am not very with that flag anyway, as the default is true so you need to write `--no-fk=false`. 

What about making this `--fk`, keeping the default as false, allowing to force FK checks (effectively this just wraps all queries in `foreign_key_checks`, for postgre it does something similar via `session_replication_role`, so adding `--fk` would disable that).

**Comment by B4nan** [comment]
Well maybe the whole flag is useless. The point was to have defaults that just works. And I think you usually need to disable FK checks to be able to adjust the schema (obviously not when you don't change any relationships). I just wanted to allow users to disable this wrapping of schema queries.

**Comment by B4nan** [comment]
Mmm btw now I understand what's happening, `yargs` consider flags with `no-` prefix as negation. So `--no-fk` is the same as `--fk=false`. 

Renamed the flag to `fk-checks`, with `false` as default and changed the description to `Do not skip foreign key checks`, so now it makes more sense I guess, as that is exactly what will happen if use this flag.

Published in `3.0.0-alpha.22`.
